src/select/__utils.ts

```diff
@@ -39,7 +32,7 @@ export function usePortalShortcut(
   });
 
   React.useEffect(() => {
-    const pickerListBox = pickerListBoxRef.current;
+    const pickerListBox = ref.current;
```

Comment:
```suggestion
    const selectMenu = ref.current;
```

`I used `pickerListBox` since that was my component, or you can give a generic name `menu`

Reply:
element?

Reply:
You copied the code from the `picker-v1` branch and there I used that variable name.

I am just asking for a better naming this component, or a generic name.

Reply:
no no i'm saying how about the variable name "element" more generic

Reply:
Oh !!!!!! 😆 

Absolutely fine